laws/src/main/scala/cats/mtl/laws/LiftValueLaws.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats
+package mtl
+package laws
+
+import cats.laws.IsEq
+import cats.laws.IsEqArrow
+import cats.syntax.apply._
+
+trait LiftValueLaws[From[_], To[_]] {
+  implicit def liftInstance: LiftValue[From, To]
+  implicit def From: Applicative[From]
+  implicit def To: Applicative[To]
+
+  // external laws:
+  def liftFPureIsPure[A](value: A): IsEq[To[A]] =
+    liftInstance.liftF(From.pure(value)) <-> To.pure(value)
+
+  def liftFTupleIsTupleLiftF[A, B](fa: From[A], fb: From[B]): IsEq[To[(A, B)]] =
+    liftInstance.liftF(From.tuple2(fa, fb)) <->
+      To.tuple2(liftInstance.liftF(fa), liftInstance.liftF(fb))
+
+  def liftFAddsNoSideEffects[A, B](a: A, toB: To[B]): IsEq[To[B]] =
+    liftInstance.liftF(From.pure(a)) *> toB <-> toB
```

Comment:
@djspiewak I've added these laws, though I'm not sure how good they are. they also seem to imply that `LiftValue` ought to contain `Applicative` instances for both of its types

Reply:
how about this for `ap`?

```scala
  def liftFApIsApLiftF[A, B](fa: From[A], ff: From[A => B]): IsEq[To[B]] =
    lift.liftF(From.ap(ff)(fa)) <-> To.ap(lift.liftF(ff))(lift.liftF(fa))
```